Digipost.Signature.Api.Client.Core/ClientConfiguration.cs

```diff
@@ -7,22 +6,25 @@ namespace Digipost.Signature.Api.Client.Core
 {
     public class ClientConfiguration
     {
-        public ClientConfiguration(Uri signatureServiceRoot, Sender sender, string certificateThumbprint)
-            : this(signatureServiceRoot, sender, CertificateUtility.SenderCertificate(certificateThumbprint, Language.English))
+        public ClientConfiguration(Environment environment, Sender sender, string certificateThumbprint)
+            : this(environment, sender, CertificateUtility.SenderCertificate(certificateThumbprint, Language.English))
         {
         }
 
-        public ClientConfiguration(Uri signatureServiceRoot, Sender sender, X509Certificate2 certificate)
+        public ClientConfiguration(Environment environment, Sender sender, X509Certificate2 certificate)
         {
-            SignatureServiceRoot = signatureServiceRoot;
+            Environment = environment;
             Sender = sender;
             Certificate = certificate;
+            
         }
 
-        public Uri SignatureServiceRoot { get; internal set; }
+        public Environment Environment { get; }
 
         public Sender Sender { get; internal set; }
 
         public X509Certificate2 Certificate { get; internal set; }
+
+        public string ServerCertificateOrganizationNumber { get; } = "SERIALNUMBER=984661185";
```

Comment:
The reason it's done in that way in the Java code is to give us the flexibility to change to a Commfides-certificate (as far as I remember). Not really likely to happen, but "tackle it once it happens" would be a bit difficult if clients have started using the library and we want to change the certificate... :cry: